great_expectations/compatibility/google.py

```diff
@@ -6,6 +6,11 @@
     "google cloud storage components are not installed, please 'pip install google-cloud-storage google-cloud-secret-manager'"
 )
 
+try:
+    import google
```

Comment:
Same question -- if it is not used for the direct `google.<something>`, then maybe we do not need it at top level?

Reply:
Good call, removed 🙇